cpp/src/bitmask/null_mask.cu

```diff
@@ -196,4 +226,26 @@ cudf::size_type count_unset_bits(bitmask_type const *bitmask, size_type start,
   return detail::count_unset_bits(bitmask, start, stop);
 }
 
+// Create a bitmask from a specific range
+rmm::device_buffer copy_bitmask(
+    bitmask_type const * mask, size_type begin_bit, size_type end_bit,
+    cudaStream_t stream, rmm::mr::device_memory_resource *mr) {
+  CUDF_EXPECTS(begin_bit >= 0, "Invalid range.");
+  CUDF_EXPECTS(begin_bit <= end_bit, "Invalid bit range.");
+  rmm::device_buffer dest_mask;
+  auto length = bitmask_allocation_size_bytes(end_bit - begin_bit);
+  if (begin_bit == 0) {
+    dest_mask = rmm::device_buffer{static_cast<void const *>(mask),
+      length, stream, mr};
+  } else {
+    dest_mask = rmm::device_buffer{length, stream, mr};
+    cudf::util::cuda::grid_config_1d config(length/sizeof(bitmask_type), 256);
```

Comment:
I am using this instead because I need ceil div.

`    auto number_of_mask_words = cudf::util::div_rounding_up_safe(
        static_cast<size_t>(end_bit - begin_bit), detail::size_in_bits<bitmask_type>());
`